src/backend/tests/unmocked/recruitment.test.ts

```diff
@@ -1,18 +1,60 @@
 import RecruitmentServices from '../../src/services/recruitment.services';
 import { AccessDeniedAdminOnlyException, HttpException } from '../../src/utils/errors.utils';
 import { batmanAppAdmin, wonderwomanGuest, supermanAdmin } from '../test-data/users.test-data';
-import { createTestOrganization, createTestUser, resetUsers } from '../test-utils';
+import { createTestFAQ, createTestOrganization, createTestUser, resetUsers } from '../test-utils';
 
 describe('Recruitment Tests', () => {
   let orgId: string;
   beforeEach(async () => {
     orgId = (await createTestOrganization()).organizationId;
+    await createTestFAQ(orgId, 'faq123');
```

Comment:
i owuld say to just add this line in the test you need it because otherwise were creating a test faq each time when sometimes we don't even need it